README.md

```diff
@@ -1,22 +1,22 @@
-## [PORTFOLIO](https://khaletska.github.io/portfolio/)
+### You can find my full portfolio [here](https://khaletska.github.io/portfolio/)
 
-### 1. About me:
-- Full-stack development student at [kood/Johvi](https://kood.tech/).
+### About me:
+- Full-stack development student at [kood/Jõhvi](https://kood.tech/).
 - iOS Intern at [Pipedrive](https://www.pipedrive.com/).
 - Passionate about turning stunning designs into seamless user experiences.
 - Technical mind with over 5 years of experience in various engineering roles.
 - Equally able to work independently and as a collaborative team player. 
 
-### 2. Favorite tools:
-- Mobile: [Swift](https://www.swift.org/)
+### Favorite tools:
+- Mobile: [Swift](https://www.swift.org/), [Objective-C](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html)
 - Front-end: [TypeScript](https://www.typescriptlang.org/), [React.js](https://react.dev/)
 - Back-end: [Go (Golang)](https://go.dev/), [Node.js](https://nodejs.org/en)
 - Database: [SQLite](https://sqlite.org/index.html)
 - Version control: [Git](https://git-scm.com/)
 - Project management: [Jira](https://jira.atlassian.com/)
 - Always ready to dive into new technologies to ensure the best outcomes for all
 
-### 3. Education:
-- Full-stack development student at kood/Johvi
+### Education:
+- Full-stack development student at kood/Jõhvi
 - Master's degree in Space Research
-- Bachelor's degree in Mechanical Egineering
+- Bachelor's degree in Mechanical Engineering
```

Comment:
this is the only change I was going to propose 😥 and then didn't manage to stop 🫠